ext/AbstractFFTsChainRulesCoreExt.jl

```diff
@@ -109,18 +119,23 @@ end
 function ChainRulesCore.rrule(::typeof(brfft), x::AbstractArray, d::Int, dims)
     y = brfft(x, d, dims)
 
-    # compute scaling factors
     halfdim = first(dims)
     n = size(x, halfdim)
 
     project_x = ChainRulesCore.ProjectTo(x)
     function brfft_pullback(ȳ)
-        dX = rfft(real.(ChainRulesCore.unthunk(ȳ)), dims) .* 2
-        selectdim(dX, halfdim, 1) ./= 2
+        dX = rfft(real.(ChainRulesCore.unthunk(ȳ)), dims)
+        # apply scaling
+        dX_scaled = similar(dX)
+        dX_scaled .= dX
+        dX_scaled .*= 2
+        v = selectdim(dX_scaled, halfdim, 1)
```

Comment:
maybe a comment above this line saying something like like `# assign view to separate variable before assignment to support Julia <1.2`?